modules/serverless-sinkbinding-yaml.adoc

```diff
@@ -0,0 +1,172 @@
+    apiVersion: batch/v1
+    kind: Job <1>
+    selector:
+      matchLabels:
+        app: heartbeat-cron
+
+  sink:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+----
++
+<1> In this example, any Job with the label `app: heartbeat-cron` will be bound to the event sink.
++
+After you have created the `sinkbinding.yaml` file, apply it by entering the following command:
++
+----
+$ oc apply --filename sinkbinding.yaml
+----
+
+. Create a CronJob.
++
+You can do this by creating a file named `heartbeats-cronjob.yaml` and copying the following sample code into it:
++
+----
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: heartbeat-cron
+spec:
+spec:
+  # Run every minute
+  schedule: "* * * * *"
+  jobTemplate:
+    metadata:
+      labels:
+        app: heartbeat-cron
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: single-heartbeat
+              image: quay.io/openshift-knative/knative-eventing-sources-heartbeats:v0.13.2
+              args:
+                - --period=1
+              env:
+                - name: ONE_SHOT
+                  value: "true"
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+----
++
+After you have created the `heartbeats-cronjob.yaml` file, apply it by entering the following command:
++
+----
+$ oc apply --filename heartbeats-cronjob.yaml
+----
+
+. Check that the controller is mapped correctly by entering the following command and inspecting the output:
++
+----
+$ oc get sinkbindings.sources.knative.dev bind-heartbeat -oyaml
+----
++
+The output should be similar to:
++
+----
+spec:
+  sink:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+      namespace: default
+  subject:
+    apiVersion: batch/v1
+    kind: Job
+    namespace: default
+    selector:
+      matchLabels:
+        app: heartbeat-cron
+----
+
+.Verification steps
+
+You can verify that the Kubernetes events were sent to the Knative event sink by looking at the message dumper function logs.
+
+You can view the message dumper function logs by entering the following commands:
+----
+$ oc get pods
+$ oc logs $(oc get pod -o name | grep event-display) -c user-container
+----
+
+The logs should contain lines similar to the following:
+----
+☁️  cloudevents.Event
+Validation: valid
+Context Attributes,
+  specversion: 1.0
+  type: dev.knative.eventing.samples.heartbeat
+  source: https://knative.dev/eventing-contrib/cmd/heartbeats/#event-test/mypod
+  id: 2b72d7bf-c38f-4a98-a433-608fbcdd2596
+  time: 2019-10-18T15:23:20.809775386Z
+  contenttype: application/json
+Extensions,
+  beats: true
+  heart: yes
+  sink: bound
```

Comment:
we dropped this override in the event-display yaml so we can drop `sink: bound` here as well :)